### ISSUE TITLE
Incorrect scoring on summary view

### ISSUE THREAD

**Issue by mikeage** [rmick/Combobulator#5]
The summary view here is correct, but the grid view shows incorrect scores (and, I think, rankings)
![summary_view](https://user-images.githubusercontent.com/87874/48327763-2efaf580-e649-11e8-965f-35f25a291174.jpg)
![grid_view](https://user-images.githubusercontent.com/87874/48327764-2f938c00-e649-11e8-93b9-052b18aa5b05.jpg)

**Comment by rmick** [comment]
Mike, was this game hosted in LTAR or normal mode?

**Comment by mikeage** [comment]
I don't have any LTAR devices. I'll double check tonight and make sure I didn't switch it by mistake, but I'm 99% sure it was normal ;-)

**Comment by rmick** [comment]
Also which do you think is right?
I think the 2nd photo is correct and the first photo is bogus.

**Comment by mikeage** [comment]
Agreed.

**Comment by rmick** [comment]
How on earth did you get a ranking of 1 and 24?

**Comment by rmick** [comment]
My head hurts !!!

I think this might be a boundary issue with the old 0/1 offset bug.

Mike please try again using players 2+3 rather than 1+2 and see what happens.

**Comment by mikeage** [comment]
Yeah, 1 vs 24 was my first sign that something was wrong. ;-) . But the scoring is also totally wrong, of course.

[Alternatively, as my kids would have put it; I came in second; you came in second from the last! :-p]

I'll try 2 + 3 tonight and let you know

**Comment by mikeage** [comment]
Good guess. 2 and 3 worked fine.

**Comment by rmick** [comment]
You are correct.
The array has 25 elements (0 is global + 24 players).
It all used to be hard-coded all through the software, but I am changing to a #define (yes I know it should be a const int, but old habits are hard to break) so that I can increase the number of players in the future.
For some reason I used <=24 in some places and <25 in others, which was dumb. This one confused me when I 'fixed' it so I ended up with <=MAX+1 - which is definitely wrong.
Note to self - do not write code whilst helping a sick child go to sleep.
